service/docs/source/requires.rst

```diff
@@ -1,24 +1,19 @@
-
 Guide to Requirements
 =====================
 
-There are several external dependencies that GEOPM requires to enable
-certain features.  The requirements for the GEOPM Service features are
-all provided by commonly used Linux distributions.  A user that is
-only interested in using the GEOPM Service will find all of the
-dependencies in this page.  A user of the GEOPM HPC Runtime should
-refer to the documentation for that feature
-`here <https://geopm.github.io/runtime.html>`__ to learn
-about the external dependencies that the runtime requires.
+The GEOPM library depends on several external dependencies for enabling
+certain features. However, most of them are readily available in standard Linux
+distributions. If you intend to use only the GEOPM Service, all the required
+dependencies can be found on this page. For users of the GEOPM HPC Runtime,
+refer to its dedicated documentation
+`here <https://geopm.github.io/runtime.html>`__ for the necessary external dependencies.
 
 Build Requirements
 ------------------
 
-There are several packages that are required to run the GEOPM service
-build.  These packages are available from standard Linux distributions.
-The following commands can be used to install them using several RPM
-based Linux distributions.
-
+To run the GEOPM service build, the packages listed below are required. All these
+packages are available in standard Linux distributions and can be installed
+using commands specific to RPM-based Linux distributions.
```

Comment:
I think io-uring-devel is missing, and there should be a note about debian package support.